src/client/src/Domains/Messages/GetMessages/index.tsx

```diff
@@ -44,6 +44,14 @@ function Messages({ receiver }: MessagesProps) {
         });
     }, [receiver, user]);
 
+    // Scroll to the bottom of the chat area for every messages update.
+    React.useEffect(() => {
+        var messageBody = document.querySelector('#chatArea');
```

Comment:
I think "chatArea" would be a better name for the "messageBody" variable

Reply:
Done.